### ISSUE TITLE
Adapt the Describe > One Variable > Frequencies as no output now from the stem option

### ISSUE THREAD

**Issue by rdstern** [IDEMSInternational/R-Instat#7980]
@N-thony I would like you to manage who does what on the set of issues for the new output window.
This one looks messy and needs a bit of detective work, to sort it out, that may be additional and unrelated.  I wonder if @Vitalis95 might be able to undertake it?  I expect others to be simpler and you may be able (in parallel) to do some yourself, in just a few minutes, or to allocate them.

This should be an easy one, but it may turn out to be a bit tricky.  It didn't work at all well for me just now.  I suspect what I have been finding may not be related to the main issue, but you may wish to correct it (them) first.

So first explore the current dialogue.
Here is what I did and the default task.  But read all below, because I then had (possibly unrelated) problems
a) Opened the usual survey data.
b) Tried the simplest option, namely all variables with the Table option, output to `Text (Output Window)`
c) Then tried the alternative HTML (Browser).
d) That's the simple task, namely the Browser option works fine and goes to a browser.  html can now go to the output window.  
e) So that's the main task.  And the change in code should be the same as in the Prepare > Check Data > View/Delete Labels that has already been done.  
f) So the html should not say (Browser) any more, and the Text should not say Output Window, because everything goes to the output window. 
g) I then tried a graph - with all the variables.  And it went straight to the R-viewer - which needs to change, because we don't use it often.  It was also an awful graph, but I suspect that should be left for now.
h) Then I tried stem and leaf for the yields - which is different and which didn't work.  I think there is some terrible programming here.
i) Then I restarted R-Instat and tried the first Table option.  Then I did stem and leaf again and it works fine.
j) But then when I return to the Table option for all variables it gives a totally different set of outputs!
k) I also tried the Both option and it now wanted to save (by default) both the graph and the table.  And it did save the table into another data frame - but not very well - though I asked it not to!

This may take too long - in which case leave it for now, but I hope not!  Or you may find you can quickly fix that Table and Stem parts - in which case disable the graph and Both options for now!

**Comment by rdstern** [comment]
@N-thony I think the above may now already be fixed.  I'll perhaps check, but alternatively it is a good dialog for one of the team involved in maths camps to check.  

But the real problem is that I now don't get anything in the output window from the Stem and Leaf plot.  Please could a developer look at that.  Maybe @Fidel365?

Also, as it is a simple output I wonder if the code could be extends, so the single receiver could become a multiple receiver.  There isn't a particular reason - perhaps - not to process multiple variables?

**Comment by N-thony** [comment]
@Fidel365 can you investigate on why we can't get any output from the Stem and Leaf Plot? Also, the possibility to change the single to a multiple receiver?

**Comment by N-thony** [comment]
> @Fidel365 can you investigate on why we can't get any output from the Stem and Leaf Plot? Also, the possibility to change the single to a multiple receiver?

@Fidel365 I found where the issue is with `Stem and Leaf Plot`. Let me know once you are ready, we can have a chat on this.

**Comment by lilyclements** [comment]
@rdstern -

@Fidel365 and I had a look at using the `stem` function in R to multiple columns. This can be done with the use of `purrr::map`:

```
# Load in data
data(mtcars)

# Define the columns of interest
columns <- c("cyl", "mpg")

# Calculate the stem of cyl and mpg using purrr::map
result <- purrr::map(columns, ~ stem(mtcars[[.x]]))
```

Perhaps this is something which should be offered when we have our "select" up and ready, since the "select" will be using `purrr::map` in a similar way to repeat the function for multiple columns. What do you think?

**Comment by rdstern** [comment]
@lilyclements that's absolutely right.  In this case I found that all the other options with this dialog had a multiple receiver and it then seemed a bit restrictive (unless we have to) to have a single receiver here, when the output was pretty simple.

**Comment by lilyclements** [comment]
@rdstern that's a good point. We have worked out the R code to apply what you would like to do. Do you think it should be implemented now?

R code - 
```
result <- purrr::map(c(COLUMN NAMES IN QUOTES), ~ stem(DATAFRAMENAME[[.x]]))
```

E.g. for columns `cyl` and `mpg` in the `mtcars` data:
```
result <- purrr::map(c("cyl", "mpg"), ~ stem(mtcars[[.x]]))
```
